### PR TITLE
[#1567] added ui classes for viewing form instance

### PR DESCRIPTION
**Pull request by valllllll2000**
#### Before the PR (what is the issue or what needed to be done)
The way we display forms is inefficient:
Manual creation of views, not using recyclerview
We don't specify which/how many fragments are kept in memory (in case of many groups)
This does not matter if there are few questions or groups but it starts to be noticeable when many groups/many questions especially repeatable ones.
Database read/write from activity and in UI Thread
#### The solution
This the first part of many others to display form which have already been exported (view only, the easier part)
Adding a few basic UI classes
Rename some constants to start on the right foot with our naming (Try using Form/Survey not SurveyGroup)
#### Screenshots (if appropriate)
NA
#### Reviewer Checklist
* [ ] Added an explanation about the work done
* [ ] Connected the PR and the issue on Zenhub
* [ ] Added a test plan to the issue
* [ ] Updated the copyright header
* [ ] Formatted the code per our style guide
* [ ] Added a documentation (if relevant)

### CODE REVIEW

File: app/src/main/java/org/akvo/flow/presentation/form/view/FormViewActivity.kt [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        val viewComponent =
+            DaggerViewComponent.builder()
+                .applicationComponent((application as FlowApp).applicationComponent)
+                .build()
+        viewComponent.inject(this)
+    }
+
+    override fun onResume() {
+        super.onResume()
+        val surveyGroup = intent.getSerializableExtra(ConstantUtil.SURVEY_EXTRA) as SurveyGroup
```
Comment (valllllll2000) [inline app/src/main/java/org/akvo/flow/presentation/form/view/FormViewActivity.kt]: leaving this one for now as renaming the class would make the change in too many places

File: app/src/main/java/org/akvo/flow/presentation/form/view/FormViewActivity.kt [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+    override fun onCreateOptionsMenu(menu: Menu): Boolean {
+        menuInflater.inflate(R.menu.view_form_activity, menu)
+        return super.onCreateOptionsMenu(menu)
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        when (item.itemId) {
+            R.id.edit_lang -> {
+                //displayLanguagesDialog()
```
Comment (iperdomo) [inline app/src/main/java/org/akvo/flow/presentation/form/view/FormViewActivity.kt]: Is this going to be implemented?

Comment (valllllll2000) [reply app/src/main/java/org/akvo/flow/presentation/form/view/FormViewActivity.kt]: it will in future PRs

**Comment by valllllll2000** [comment]
@iperdomo in the end not sure if you approved it or not, please confirm

gif

**Comment by iperdomo** [comment]
@valllllll2000 it is approved, I use the wrong button when I commented one line

**Pull request merged**